ldjapic/TravAlarm
# Use Expo Template

Using expo CLI, I created a new template to use:
```
expo init --template expo-template-blank@sdk-36
```

This PR removes the old template files and replaces them with the files generated by expo. Husky was also added for future use (ie linting pre commit hook). README was also updated with local setup steps.

TEST STEPS:
- Checkout branch locally 
- Follow the local setup steps
- Confirm the app runs in expo without any errors
